dspace-api/src/main/java/org/dspace/app/util/DCInput.java

```diff
@@ -505,4 +528,12 @@ public boolean validate(String value) {
 
         return true;
     }
+
+    public boolean isRelationshipField() {
+        return relationshipField;
+    }
+
+    public boolean isMetadataField() {
+        return metadataField;
```

Comment:
While these methods (`isRelationshipField` and `isMetadataField`) are simple getters, I think we should document what these *mean*.  For instance, it's not immediately clear whether a single field can be BOTH a metadataField and a relationshipField (I think it is possible though, based on the later code).  This could be cleared up via JavaDocs on these methods

Reply:
The JavaDocs has been included